src/libraries/System.Private.Xml/src/System/Xml/Serialization/Compilation.cs

```diff
@@ -69,19 +71,25 @@ internal TempAssembly(XmlMapping[] xmlMappings, Type?[] types, string? defaultNa
                     _assembly = GenerateRefEmitAssembly(xmlMappings, types);
                 }
                 // Only catch and handle known failures with RefEmit
-                catch (CodeGeneratorConversionException)
+                catch (CodeGeneratorConversionException ex)
                 {
-                    fallbackToCSharpAssemblyGeneration = true;
+                    // There is no CSharp-generating/compiling fallback in .Net Core because compilers are not part of the SDK.
```

Comment:
```suggestion
                    // There is no CSharp-generating/compiling fallback in .Net Core because compilers are not part of the runtime.
```
Compilers are part of the .NET SDK. The problem is they are not part of the runtime.